server/apps/distributed-app/docs/modules/ROOT/pages/extending/smtp-hooks.adoc

```diff
@@ -1,7 +1,7 @@
 = Distributed James Server &mdash; Custom SMTP hooks
 :navtitle: Custom SMTP hooks
 
-SMTP hooks enable extending capabilities of the SMTP server and are run synchronously upon email reception, before the email is
+SMTP hooks enable extening capabilities of the SMTP server and are run synchronously upon email reception, before the email is
```

Comment:
Previous version was the correct one, put back `extending` please :)

Reply:
Previous version was the correct one, put back extending please :)